pkg/release/push.go

```diff
@@ -316,25 +336,37 @@ func (p *PushBuild) StageLocalArtifacts(version string) error {
 	}
 
 	// Copy helpful GCP scripts to local GCS staging directory for push
```

Comment:
nitpick: should we move this comment inside conditional block since the GCP script copy is 1 of 2 steps that will take place if `util.Exists(extraPath)`?

Reply:
Yes, fixed as suggested :+1: